suite-native/module-onboarding/src/screens/BiometricsScreen.tsx

```diff
@@ -0,0 +1,106 @@
+import { useDispatch } from 'react-redux';
+
+import { CommonActions } from '@react-navigation/core';
+
+import { setIsOnboardingFinished } from '@suite-native/settings';
+import {
+    HomeStackRoutes,
+    OnboardingStackParamList,
+    OnboardingStackRoutes,
+    RootStackRoutes,
+    Screen,
+    ScreenHeader,
+    StackProps,
+} from '@suite-native/navigation';
+import { Box, Button, HStack, Text, VStack } from '@suite-native/atoms';
+import { Translation } from '@suite-native/intl';
+import { EventType, analytics } from '@suite-native/analytics';
+import { BiometricsSvg, useBiometricsSettings } from '@suite-native/biometrics';
+import { Icon } from '@suite-native/icons';
+
+export const BiometricsScreen = ({
+    navigation,
+}: StackProps<OnboardingStackParamList, OnboardingStackRoutes.Biometrics>) => {
+    const { toggleBiometricsOption } = useBiometricsSettings();
+
+    const dispatch = useDispatch();
+
+    const enableBiometrics = async () => {
+        const result = await toggleBiometricsOption();
+        if (result === 'enabled') {
+            analytics.report({
+                type: EventType.BiometricsChange,
+                payload: {
+                    enabled: true,
+                    origin: 'bottomSheet',
+                },
+            });
+        }
+    };
+
+    const exitOnboardingFlow = () => {
+        dispatch(setIsOnboardingFinished());
+
+        navigation.dispatch(
+            CommonActions.reset({
+                index: 0,
+                routes: [
+                    {
+                        name: RootStackRoutes.AppTabs,
+                        params: {
+                            screen: HomeStackRoutes.Home,
+                        },
+                    },
+                ],
+            }),
+        );
+    };
+
+    const handleEnableButtonPress = async () => {
+        await enableBiometrics();
+        exitOnboardingFlow();
+    };
+
+    const handleNotNowButtonPress = () => {
+        exitOnboardingFlow();
+    };
+
+    return (
+        <Screen header={<ScreenHeader />}>
+            <VStack justifyContent="space-between" flex={1} alignItems="center">
```

Comment:
The `alignItems` property causes incorrect content padding.
```suggestion
            <VStack justifyContent="space-between" flex={1}>
```

Before and after:
<img width="355" alt="image" src="https://github.com/user-attachments/assets/fdb9a17f-428d-4422-8368-6f3a5769e95d" />
<img width="355" alt="image" src="https://github.com/user-attachments/assets/4cff00e5-8b24-4215-9a29-f364fce2b0ea" />

Reply:
good catch, fixed 212491054b